### DISCUSSION TITLE
Dispatchable hydro-power

### DISCUSSION THREAD

**Discussion by Wuhochi** [Critical-Infrastructure-Systems-Lab/PowNet#13] [Q&A] [answered]
Dear Sir,

I'm appreciate of your valuable work. And I would like to ask some questions, as you mentioned in earth's future paper, the current hydropower output is modeled as external in PowNet.

1) However, in reality, some hydropower stations has a cascade relationship, and they can also be dispatched to meet better operational cost.

2) Would you consider adding capacity and transmission line expansion model to the model in the future?

Thank you very much.
Best regards,
haochi

**Comment by stefano-galelli** [comment, marked as answer]
Dear Haochi,

Thanks for your interest in PowNet! Let me answer your questions:

1. Yes, this is a very good point. From a modelling point of view, one typically needs to choose between modelling these cascading effects in the hydrological model or in the power system model. At this stage, our preference is to model cascading effects in the hydrological model (we use VIC-Res --> https://github.com/thanhiwer/VICRes), because this allows us to account for many other hydrological processes. However, the limitation is that an hydrological model only calculates the available hydropower (which is an input to PowNet). Representing such cascading effects in a power system model would require to simplify the representation of the hydrological process. In either case, there is an underlying model simplification that must be adopted. To solve the problem, one needs, in my opinion, to hard-couple the hydrological and power system model, that is, creating a two-way exchange of information between the models. We have implemented it (for the case study of Cambodia) and our paper is currently under review. We can share the code with you as soon as the paper is published. You can contact me at stefano_galelli@sutd.edu.sg if you need additional info.

2. This is not planned at the moment. Two capacity expansion models I can recommend (both available on GitHub) are urbs and GridPath.  

Best,
Stefano

**Reply by Wuhochi** [reply]
Thank you for giving me this valuable insight.  Hope to have further communications with you : )
